### ISSUE TITLE
Laravel 5: Unable to prepare route [__clockwork/{id}] for serialization. Uses Closure.

### ISSUE THREAD

**Issue by poul-kg** [itsgoingd/clockwork#99]
When I run

`php artisan route:cache`

I receive the following error:

```
 [LogicException]
  Unable to prepare route [__clockwork/{id}] for serialization. Uses Closure.
```

**Comment by itsgoingd** [comment]
Hey, this is already fixed in 1.x-dev, I'll tag a new stable release in a week or so.

**Comment by itsgoingd** [comment]
Fixed in Clockwork 1.8.
